### PR TITLE
EE: PLT-4512 Show secret in addition to QR code when activating MFA

### PR DESCRIPTION
**Pull request by jwilander**
#### Summary
Add displaying of MFA secret to activation screen and add a unit test. Thanks to @asaadmahmood for the UI work.

#### Ticket Link
https://mattermost.atlassian.net/browse/PLT-4512

#### Checklist
- [x] Added or updated unit tests (required for all new features)
- [x] All new/modified APIs include changes to the drivers
- [x] Has enterprise changes (mattermost/enterprise#65)
- [x] Has UI changes
- [x] Includes text changes and localization file ([.../i18n/en.json](https://github.com/mattermost/platform/blob/master/i18n/en.json) and [.../webapp/i18n/en.json](https://github.com/mattermost/platform/tree/master/webapp/i18n/en.json)) updates
- [x] Touches critical sections of the codebase (auth, upgrade, etc.)

### CODE REVIEW

**Comment by lfbrock** [comment]
+1, tested setting up manually using Google Authenticator on iPhone 5s and it worked.

File: Makefile [hunk 1]
```diff
@@ -204,6 +204,7 @@ ifeq ($(BUILD_ENTERPRISE_READY),true)
 
 	$(GO) test $(GOFLAGS) -run=$(TESTS) -covermode=count -c ./enterprise/ldap && ./ldap.test -test.v -test.timeout=120s -test.coverprofile=cldap.out || exit 1
 	$(GO) test $(GOFLAGS) -run=$(TESTS) -covermode=count -c ./enterprise/compliance && ./compliance.test -test.v -test.timeout=120s -test.coverprofile=ccompliance.out || exit 1
+	$(GO) test $(GOFLAGS) -run=$(TESTS) -covermode=count -c ./enterprise/mfa && ./mfa.test -test.v -test.timeout=120s -test.coverprofile=cmfa.out || exit 1
```
Comment (crspeller) [inline Makefile]: Added it here but wrote no tests?

Comment (jwilander) [inline Makefile]: Test is here :P https://github.com/mattermost/enterprise/pull/65/files

Comment (crspeller) [inline Makefile]: Oh ya. Duh. I don't see the enterprise changes link when its burred in that list.

**Pull request merged**